.github/workflows/ci.yml

```diff
@@ -84,6 +84,18 @@ jobs:
           --mount type=bind,source="${GITHUB_WORKSPACE}",target=/project \
           splunk-otel-dotnet/${{ matrix.base-image }} \
           ./build.sh Workflow
+    - name: Test the Shell scripts from README.md in Docker container
+      run: |
+        set -e
+        docker build -t mybuildimage -f "./build/docker/${{ matrix.base-image }}.dockerfile" .
```

Comment:
I suggest moving `./build/docker` to `./docker` to have the same repo structure as upstream.